models/_sector/nft/trades/ethereum/platforms/_schema.yml

```diff
@@ -312,6 +312,34 @@ models:
            - price_raw
            - platform_fee_amount_raw
            - royalty_fee_amount_raw
+
+ - name: looksrare_seaport_ethereum_base_trades
+   meta:
+     blockchain: ethereum
+     sector: nft
+     project: looksrare
+     contributors: [ 'hildobby' ]
+   config:
+     tags: [ 'ethereum', 'nft', 'trades', 'looksrare' ]
+   description: "looksrare's seaport base trades"
+   tests:
```

Comment:
rob introduced a new test, as seen here:
https://github.com/duneanalytics/spellbook/blob/main/models/_sector/nft/trades/ethereum/platforms/_schema.yml#L13-L14

which you can apply here by simply adding that one row. you shouldn't need to do anything else. it will check that the columns written are the correct data type for `nft.trades` lineage